### ISSUE TITLE
Tests are broken due to failing transip provider tests

### ISSUE THREAD

**Issue by pavantc** [github/octodns#512]
Looks like the transip resources are down causing breakages in the tests:

```
...........................................................................................................................................................................................................................................................EEEE..........................................................................................
======================================================================
ERROR: test_apply (test_octodns_provider_transip.TestTransipProvider)
----------------------------------------------------------------------
```

http://www.transip.nl/soap seems to be inaccessible.

**Comment by olafz** [comment]
TransIP changed their API on March 5th, 2020: https://www.transip.nl/nieuws/de-nieuwe-transip-rest-api-is-live/

> Voor onze nieuwe API zijn we overgestapt van SOAP naar het REST-protocol. Dit houdt onder andere in dat de API voortaan JSON in plaats van XML returnt

Translation: the new API moved from SOAP to REST. This means - for example - that the API now returns JSON instead of XML.

**Comment by ross** [comment]
Thanks @olafz that's kind of what it looked like. Unfortunately it seems the most recent transip package in pypy is 1 year old and appears to use the soap interface https://pypi.org/project/transip/#history.

**Comment by olafz** [comment]
@ross The repository itself looks kind-of alive though: https://github.com/benkonrath/transip-api. I just cross-referenced an issue there (the same cause); let's see if anything happens. Otherwise I might take a look myself when I have a bit more time.

**Comment by benkonrath** [comment]
I just released a new version (2.1.0) that should fix the issue (thanks to some help). You can try this version but the TransIP SOAP API has been thankfully deprecated and will be turned off at some point (I guess).
